spec/ttt/board_spec.rb

```diff
@@ -1,18 +1,21 @@
 require_relative '../../ttt/src/board'
 require_relative '../../ttt/src/player'
-
+require_relative '../../ttt/src/errors/invalidBoardSize'
```

Comment:
file names in ruby should be snake cased.

Reply:
👍